components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -2363,6 +2363,7 @@ private ConfigParameters() {
         public static final String GATEWAY_INSTRUCTION_REMOVE = "Remove";
         public static final String SYNAPSE_ARTIFACTS = "/synapse-artifacts";
         public static final String SYNAPSE_ATTRIBUTES = "/synapse-attributes";
+        public static final String INTERNAL_BASE_URL = "/internal/data/v1";
```

Comment:
Can't we use the existing INTERNAL_WEB_APP_EP rather than defining a new.

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/9001/commits/b35c9cc5bb0814659fffcbd381d356b54d62f04f